src/llmcompressor/transformers/finetune/runner.py

```diff
@@ -222,6 +225,7 @@ def run_sequential_stages(
                     recipe_stage=stage_name,
                 )
             elif run_type is StageRunType.TRAIN:
+                self.trainer.model = model
```

Comment:
This also does not affect this example, but makes model passing slightly clearer

Reply:
Oh I see